### PR TITLE
[Cloud Security]  auto import for ecs fields

### PR DESCRIPTION
**Pull request by CohenIdo**
solves https://github.com/elastic/security-team/issues/6471

using `import_mappings: true` for auto using ecs fields.

### CODE REVIEW

**Comment by amirbenun** [comment]
I guess you can also delete the vulnerabilities `ecs.yml`, right?

**Comment by CohenIdo** [comment]
> I guess you can also delete the vulnerabilities `ecs.yml`, right?

thanks @amirbenun

**Comment by CohenIdo** [comment]
move back to draft as all the task is blocked during [a bug was found](https://github.com/elastic/security-team/issues/6471#issuecomment-1560700135) in package upgrade

File: packages/cloud_security_posture/manifest.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 format_version: 2.3.0
 name: cloud_security_posture
 title: "Security Posture Management"
-version: "1.4.0-preview2"
+version: "1.2.12"
```
Comment (kfirpeled) [inline packages/cloud_security_posture/manifest.yml:4]: @CohenIdo  pay attention when testing it locally that 1.2.13 is our latest version - so you might not work with the version you expect it to be

**Pull request closed without merging**